### PR TITLE
Dev lukas

### PR DESCRIPTION
**Pull request by aut0**

### CODE REVIEW

**Comment by aut0** [comment]
@barnasc I updated the file as discussed in the meeting. It's ready for review.

File: RELEASENOTES.md [hunk 1]
```diff
@@ -24,60 +24,14 @@ Supported platforms and tool dependencies
...
-
-  * GNU C++ compiler versions prior to 4.2.x (all platforms)
-  * FreeBSD 9.0 or later (x86, x86_64) with GNU C++ compiler
-  * HP-UX 11.00 or later with GNU C++ or HP C++ compiler
-  * Sun/Oracle Solaris, with GNU C++ or Sun/Solaris Studio compiler
-  * Mac OS X prior to 10.12 (Sierra) and/or on the x86, PowerPC architectures
-  * Microsoft Visual C++ versions prior to 10.0 (2010)
-  * Cygwin 1.7 or later (x86, x86_64) with GNU C++ compiler
-  * Msys/MinGW32 GNU C++ compiler
+This release is supported on the same platforms as the Accellera SystemC kernel.
```
Comment (barnasc) [inline RELEASENOTES.md:27]: we normally not referencing to something called kernel. We either call it proof-of-concept or reference implementation. Propose to rephase to:

This release is supported on the same platforms as the Accellera SystemC reference implementation.
These are documented in the SystemC reference implementation RELEASENOTES

File: docs/DEVELOPMENT.md [hunk 2]
```diff
@@ -308,55 +308,36 @@ performed by the maintainer
...
    and optionally signed Git tag.
 
    ```
    # git tag -a -m "<package> <version>" <version> <refspec>
-   git tag -a -m "CCI 1.0.1" 1.0.1 release
+   git tag -a -m "CCI 1.0.1" v1.0.1 release
    ```
 
-   The tagname should contain the `<version>` in `Major.Minor.Patch` format
+   The tagname should contain the `<version>` in `vMajor.Minor.Patch` format
```
Comment (barnasc) [inline docs/DEVELOPMENT.md:322]: The section _Version scheme_ in this document also needs to add the v prefix..

**Review by barnasc — COMMENTED** [review]
review comments inline

**Comment by barnasc** [comment]
additional thing seen during regression in the automake flow: 

> No rule to make target 'ChangeLog', needed by 'all-am'.  Stop.

We need to delete this `ChangeLog` entry in [Makefile.am](https://github.com/accellera-official/cci/blob/master/Makefile.am)

File: docs/DEVELOPMENT.md (hunk 2 above)
Comment (aut0) [reply docs/DEVELOPMENT.md:322]: The section above mentions the versioning scheme, which I think stays unchanged.
If my understanding is correct, the next version of SystemC CCI will be called "1.0.1", but not "v1.0.1".
It's just the tag that we name "v1.0.1".
Maybe we should discuss if we want the tag to carry the same name as the version, i.e. "1.0.1".

**Comment by aut0** [comment]
> additional thing seen during regression in the automake flow:
> 
> > No rule to make target 'ChangeLog', needed by 'all-am'.  Stop.
> 
> We need to delete this `ChangeLog` entry in [Makefile.am](https://github.com/accellera-official/cci/blob/master/Makefile.am)

Done

**Review by barnasc — APPROVED** [review]

**Comment by barnasc** [comment]
approved. Can be merged

**Pull request merged**